app/models/Status.php

```diff
@@ -56,11 +57,18 @@ public function create($name, $color, $default = 0, $team = null)
             $name = 'Unnamed';
         }
 
-        $sql = "INSERT INTO status(name, color, team, is_default) VALUES(:name, :color, :team, :is_default)";
+        if ($allowTimestamp != 'false') {
```

Comment:
There is no need for that here. Instead of $allowTimestamp being a bool, use an int, like the $default parameter and use it directly.